randovania/games/super_metroid/json_data/Wrecked Ship.txt

```diff
@@ -166,7 +162,8 @@ Extra - save_index: 0
   * Layers: default
   * Normal Door to Wrecked Ship Main Shaft/Door to Shaft Save Station
   > Save Station
-      Trivial
+      # TODO: doesnt this save station only get activated when power is on?
+      Normal Damage
```

Comment:
No damage required here. You can reach this node, but the save station will not be active until is the power is on. I thought this would be 'Event - Phantoon Defeated' but it has its own node. Not sure how its different.

Reply:
the "1 damage" was there as a placeholder so i can add the comment :)

wrecked ship being active, and phantoon being dead are seperate probably for future reasons, so that you can i.e. make ship active by defeating kraid.